Commands/Public/wolfram.js

```diff
@@ -0,0 +1,50 @@
+const fetch = require("chainfetch");
+const { tokens: { wolframAppID } } = require("../../Configurations/auth");
+
+module.exports = async ({ Constants: { Colors, APIs, UserAgent, Text } }, documents, msg, commandData) => {
+	if (!msg.suffix) {
+		return msg.send({
+			embed: {
+				color: Colors.INVALID,
+				description: Text.INVALID_USAGE(commandData, msg.guild.commandPrefix),
+			},
+		});
+	}
+
+	const response = await msg.send(Text.THIRD_PARTY_FETCH("Fetching data from Wolfram|Alpha"));
+
+	const query = encodeURIComponent(msg.suffix);
+	const body = (await fetch.get(APIs.WOLFRAM(wolframAppID, query)).set("User-Agent", UserAgent).onlyBody()).queryresult;
```

Comment:
This can throw if api errors...

Reply:
It should. It’ll get handled by the generic command handler error catcher. The message seemed fit for this type of error; unexpected and unknown.